tests/chainer_tests/datasets_tests/test_pickle_dataset.py

```diff
@@ -26,6 +58,20 @@ def test_write_read(self):
         assert dataset[2] == 1.5
         assert dataset[1] == 'hello'
 
+    def test_after_fork(self):
+        writer = datasets.PickleDatasetWriter(self.io)
+        writer.write(1)
+        writer.flush()
+
+        reader = ReaderMock(self.io)
+        _ = datasets.PickleDataset(reader)
```

Comment:
Please omit the variable.
>/repo/tests/chainer_tests/datasets_tests/test_pickle_dataset.py:67:9: F841 local variable '_' is assigned to but never used

Reply:
IMO, we cannot remove the variable.
This test require the instance to live until the child process is initialized.
If the variable is removed, the instance is destructed immediately and the test is to fail.

Reply:
Oh, I see.
So how about using other variable name instead of `_` (`dataset`?) and touching the variable at the end of the test? (like `del dataset`?)
Please also write some brief comment of the intent.